### PR TITLE
Added changes for VPN Mac signing

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

File: src/scriptworker/constants.py [hunk 1]
```diff
@@ -504,9 +505,9 @@
                 {
                     "firefox": ("decision", "action", "docker-image"),
                     "thunderbird": ("decision", "action", "docker-image"),
+                    "mozillavpn": ("decision", "action", "docker-image"),
```
Comment (hneiva) [inline src/scriptworker/constants.py:508]: @escapewindow BTW, I'm not 100% sure this is what we want. Could you please confirm.

Comment (escapewindow) [reply src/scriptworker/constants.py:508]: This is what we want. We're already pinned to a dockerhub decision image, so we should be good.
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/64d6b0147c698c8baba189b107fc09fc2ee011df/.taskcluster.yml#L241

File: src/scriptworker/constants.py [hunk 2]
```diff
@@ -67,6 +67,7 @@
         "task_log_dir": "...",  # set this to ARTIFACT_DIR/public/logs
         "artifact_upload_timeout": 60 * 20,
         "max_concurrent_downloads": 5,
+        "provisionprofile_dir": "...",  # For apple signing
```
Comment (escapewindow) [inline src/scriptworker/constants.py:70]: This looks like an iscript script_config.yml setting, not a scriptworker worker daemon config setting, no?

**Review by escapewindow — CHANGES_REQUESTED** [review]

**Comment by ahal** [comment]
Looks fine to me, but I'm not familiar with this code at all, so going to defer to Aki here.

**Comment by hneiva** [comment]
Changed how `provisionprofile_dir` is loaded. Not needed anymore.

**Pull request closed without merging**